doc/source/guidelines.rst

```diff
@@ -0,0 +1,19 @@
+.. _guidelines:
+
+Guidelines and variable naming conventions
+==========================================
+
+The variable name (code) should adhere to the following conventions:
+
+*  A *|* (pipe) character indicates levels of hierarchy.
+*  Do not use spaces before and after the *|* character, but add a
+   space between words (e.g., *Primary Energy|Non-Biomass Renewables*).
+*  Do not use abbreviations of statistical operations (*min*, *max*,
+   *avg*) but always spell out the word.
+*  All words must be capitalised (except for *and*, *w/*, *w/o*, etc.).
+*  Do not use abbreviations (e.g, *PHEV*) unless strictly necessary.
```

Comment:
I'd move this line up to line 13, so both abbreviations conventions are in sequence.

Reply:
done